test/e2e/broker_default_test.go

```diff
@@ -219,6 +222,44 @@ func TestDefaultBrokerWithManyTriggers(t *testing.T) {
 	}
 }
 
+// This test is for avoiding regressions on the trigger dependency annotation functionality.
+// It will first create a trigger with the dependency annotation, and then create a pingSource.
+// Broker controller should make trigger become ready after pingSource is ready.
+func TestTriggerDependencyAnnotation(t *testing.T) {
```

Comment:
Would it be better to move this test to a different file like `trigger_dependency_annotation_test.go`?

Reply:
I have thought it before. the reason why I still put it here is: now the trigger reconciler was combined with broker reconciler (together as a single broker controller), the trigger is somehow a default feature for broker. I can move it out side of this file if you find that is more meaningful.

Reply:
I think it makes more sense to move it out as it is still a feature for trigger. Or you can rename this file to `broker_trigger_default_test.go`? Either way, it's up to you.

Reply:
Sure, I'll move it out.